Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/rb.py

```diff
@@ -0,0 +1,259 @@
+from qm.qua import *
```

Comment:
Is this file different from rb.py in the main folder ?
If not, then I would just refer to it in the readme, what do you think?

Reply:
yes, it has the modificiation of groun dstate preparation

Reply:
Ok, what you call "ground state preparation" could actually be active reset right?

Reply:
yes indeed